### PR TITLE
provision.sh: include required trento-runner service file

### PR DESCRIPTION
**Pull request by rtorrero**
This PR installs the trento-runner service file when executing the provision.sh script to avoid a failure on a pipeline run.

### CODE REVIEW

File: hack/provision.sh [hunk 1]
```diff
@@ -101,6 +102,16 @@ install_actions_runner() {
...
   systemctl enable --now "$ACTIONS_RUNNER_SYSTEMD_UNIT"
 
+  if [ -f "/etc/systemd/system/$TRENTO_RUNNER_SYSTEMD_UNIT" ]; then
+    echo "  Warning: trento-runner systemd unit already installed. Removing..."
+    systemctl stop "$TRENTO_RUNNER_SYSTEMD_UNIT"
+    rm "/etc/systemd/system/$TRENTO_RUNNER_SYSTEMD_UNIT"
+  fi
+
+  echo "  Installing systemd unit"
+  ./svc.sh install >/dev/null
```
Comment (dottorblaster) [inline hack/provision.sh:112]: Could we separate the bracket with a space after, like `./svc.sh install > /dev/null`?

**Review by dottorblaster — DISMISSED** [review]
LGTM but have a look at the comment

**Review by arbulu89 — DISMISSED** [review]
@rtorrero We didn't actually work in the `provision.sh` script, as we thought that It would be removed, and we would have some new strategy for the CD.
I did all the installation manually for the 0.4.0.

I don't really know if it makes sense to continue with this approach, as it is not scalable at all and requires fair amount of maintenance. Anyway, I think that this discussion already happened within the team.

**Comment by rtorrero** [comment]
@arbulu89 totally agreed, but as we had some new faces joining us I wanted to have this fixed until we are able to remove it

**Comment by rtorrero** [comment]
> Could we separate the bracket with a space after, like `./svc.sh install > /dev/null`? 

sure thing

**Review by arbulu89 — APPROVED** [review]

**Comment by rtorrero** [comment]
Closing this as it doesn't make sense anymore now that the hacks folder is gone. I'm actually happy about this :-)

**Pull request closed without merging**